10up/classifai
# Ability to generate an image directly

### Description of the Change
This PR adds the **Media > Generate Images** admin menu to have a direct option to generate OpenAI DallE images.


Closes [#503](https://github.com/10up/classifai/issues/503)

### How to test the Change
- Go to Media > Generate Images
- Check if the Media Modal is working correctly


### Credits

Props @phpbits 


### Checklist:
